### ISSUE TITLE
Bug in static class Log

### ISSUE THREAD

**Issue by erikbra** [chucknorris/roundhouse#278]
Hi. I just found an error in the static `Log.bound_to` method. It returns:

```
  logger = Container.get_an_instance_of<LogFactory>().create_logger_bound_to(object_that_needs_logging?.GetType());
```

but the signature of `create_logger_bound_to` is `create_logger_bound_to(object type)`, so it expects the type of the object, not the type itself. The method should then read:

```
  logger = Container.get_an_instance_of<LogFactory>().create_logger_bound_to(object_that_needs_logging?.GetType());
```
Would you like a separate pull request for this fix, or should I just fix it as part of #187 ? Might there be functionality depending on the logger always being `Log4NetLogger`, as is done in the exception handling?

**Comment by BiggerNoise** [comment]
Ideally a separate P/R, but I'll accept the fix as part of #187. If you do a separate P/R, I'll merge it ASAP so you're not having to wait around for it.

**Comment by erikbra** [comment]
This was fixed as part of #187 . So we can close it.
